test/e2e/README.md

```diff
@@ -42,35 +67,78 @@ go test --tags=e2e ./test/e2e/... --count=3
 If you want to run a specific test:
 
 ```shell
-go test --tags=e2e ./test/e2e/... -run NameOfTest
+E2E_PROJECT_ID=<project name> \
+  go test --tags=e2e ./test/e2e/... -run NameOfTest
 ```
 
 For example, to run TestPullSubscription:
 
-```shell
-GOOGLE_APPLICATION_CREDENTIALS=<path to json creds file> \
+```shell \
 E2E_PROJECT_ID=<project name> \
   go test --tags=e2e ./test/e2e/... -run TestPullSubscription
 ```
 
-Note that if you plan on running metrics-related E2E tests using the StackDriver
-backend, you need to give your
-[Service Account](../../docs/install/pubsub-service-account.md) the
-`Monitoring Editor` role on your Google Cloud project:
+### Running E2E tests with authentication mechanism using Workload Identity
+```shell
+go test --tags=e2e 
+  -workloadIndentity=true \
+  -pubsubServiceAccount=cre-pubsub@$PROJECT_ID.iam.gserviceaccount.com \
```

Comment:
maybe pointed out the `-pubsubServiceAccount=cre-pubsub@$PROJECT_ID.iam.gserviceaccount.com` is the GSA used for the resources